administrator/components/com_categories/Service/HTML/AdministratorService.php

```diff
@@ -86,7 +86,7 @@ public function association($catid, $extension = 'com_content')
 						$classes  = 'badge badge-secondary';
 
 						$item->link = '<a href="' . $url . '" title="' . $item->language_title . '" class="' . $classes . '">' . $text . '</a>'
-							. '<div role="tooltip" id="tip' . (int) $item->id . '">' . $tooltip . '</div>';
+							. '<div role="tooltip" id="tip' . (int) $catid . (int) $item->id . '">' . $tooltip . '</div>';
```

Comment:
It's possible that this is not a unique string, you have to add a separator between the numbers like a dash.
```suggestion
							. '<div role="tooltip" id="tip-' . (int) $catid . '-' . (int) $item->id . '">' . $tooltip . '</div>';
```